python/ray/autoscaler/docker.py

```diff
@@ -46,18 +39,8 @@ def dockerize_if_needed(config):
                                             docker_mounts, cname,
                                             run_options + worker_run_options)
 
-    config["head_setup_commands"] = head_docker_start + (with_docker_exec(
-        config["head_setup_commands"], container_name=cname))
-    config["head_start_ray_commands"] = (
-        docker_autoscaler_setup(cname) + with_docker_exec(
-            config["head_start_ray_commands"], container_name=cname))
-
-    config["worker_setup_commands"] = worker_docker_start + (with_docker_exec(
-        config["worker_setup_commands"], container_name=cname))
-    config["worker_start_ray_commands"] = with_docker_exec(
-        config["worker_start_ray_commands"],
-        container_name=cname,
-        env_vars=["RAY_HEAD_IP"])
+    config["docker"]["worker_docker_start"] = worker_docker_start
+    config["docker"]["head_docker_start"] = head_docker_start
```

Comment:
is there an example that shows this?

Reply:
What do you mean? In terms of why the commands are different? The `aws/example-gpu-docker.yaml` has GPU specific commands for only the head start command.

Reply:
The API "worker_docker_start" is an 'internal' API